### PR TITLE
trash, filters implemented, 404 page, responsiveness

### PR DESCRIPTION
**Pull request by kgajriaAgarwal**
Hello, i have implemented trash and notes filter functionality using Api's, 404 page not found and also added responsiveness . kindly review it and let me know what do think about it !
files to check-
src/Helpers/Context/LayoutContext.js
src/Helpers/Context/ReducerContext.js
src/Helpers/Context/TrashContext.js
src/Helpers/Reducer/filterFunctions.js
src/Helpers/Reducer/reducer.js
src/Pages/Shared/NoMatch/NoMatch.js
src/Pages/Shared/RightBar/RightBar.js
src/Pages/Trash/Trash.js
PREVIEW : https://blithe-la-nota.netlify.app/
PR : https://github.com/kgajriaAgarwal/blithe-lanota/pull/5

### CODE REVIEW

File: src/Helpers/Context/TrashContext.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+const TrashProvider = ({ children}) => {
+
+    const [trashData, setTrashData] = useState([]);
+    const { setNotes } = useNotes();
+    let authData = getLocalStorage('authData');
+
+      ////action -add Note To Archives
+    const addNoteToTrash = async (noteId , noteObj) => {
+        console.log("noteId , noteObj", noteId , noteObj);
```
Comment (sameep-s) [inline src/Helpers/Context/TrashContext.js]: you can remove these console.log's

File: src/Pages/NotesPage/NotesPage.js [hunk 2]
```diff
@@ -36,20 +46,40 @@ const NotesPage = (props) => {
       });
   }, [notes, archives]);
 
+  // const filteredPriorites = priorityfiltering(notesData, priority);
```
Comment (sameep-s) [inline src/Pages/NotesPage/NotesPage.js]: you can remove this commented code.

**Review by sameep-s — APPROVED** [review]
Code is good but there is a lot of unused commented code and unnecessary spaces in the code you can remove it if it does'nt affect the functionality. otherwise it's good to merge.

File: src/Helpers/Reducer/filterFunctions.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    let filteredPriorities =
+      priority.length !== 0
+        ? notes.filter((p) => priority.includes(p.priority))
+        : notes;
+    return filteredPriorities;
+  };
+  
+  const getSortedDates = (notes, timeSort) => {
+    const dateToTime = (date) => {
+      const d = new Date(date);
```
Comment (yaswanthmyneni) [inline src/Helpers/Reducer/filterFunctions.js]: changes: use proper variable names.

File: src/Pages/NotesPage/CreateNote.js [hunk 4]
```diff
@@ -208,9 +208,9 @@ const CreateNote = (props) => {
               >
                 <MenuItem value="Home">Home</MenuItem>
                 <MenuItem value="Work">Work</MenuItem>
-                <MenuItem value="Personal">Personal</MenuItem>
-                <MenuItem value="Exercise">Exercise</MenuItem>
-                <MenuItem value="Chores">Chores</MenuItem>
+                {/* <MenuItem value="Personal">Personal</MenuItem> */}
+                {/* <MenuItem value="Exercise">Exercise</MenuItem> */}
+                {/* <MenuItem value="Chores">Chores</MenuItem> */}
```
Comment (yaswanthmyneni) [inline src/Pages/NotesPage/CreateNote.js]: changes: remove unused code

File: src/Pages/Shared/NoteCard/NoteCard.js [hunk 5]
```diff
@@ -79,34 +81,59 @@ export const NoteCard = (props) => {
...
-            onClick={() =>
-                isModule === "NotesPage"
-                  ? deleteNote(note._id)
-                  : deleteNoteFromArchives(note._id)
-              }
+            // onClick={() =>
+            //     isModule === "NotesPage"
+            //       ? deleteNote(note._id)
+            //       : deleteNoteFromArchives(note._id)
+            //   }
```
Comment (yaswanthmyneni) [inline src/Pages/Shared/NoteCard/NoteCard.js]: changes: here too and everywhere

File: src/Pages/Shared/RightBar/RightBar.js [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+    }
+    setChecked(newChecked);
+  };
+
+  const handleRadioChange = (event) => {
+    dispatch({
+      type: event.target.value.toUpperCase(),
+    });
+    setSortByDate(event.target.value);
+    console.log(event.target.value);
```
Comment (yaswanthmyneni) [inline src/Pages/Shared/RightBar/RightBar.js]: changes: remove logs before pushing to github

**Review by yaswanthmyneni — APPROVED** [review]
you can merge after changes

File: src/Helpers/Context/TrashContext.js (hunk 1 above)
Comment (kgajriaAgarwal) [reply src/Helpers/Context/TrashContext.js]: removed!

File: src/Helpers/Reducer/filterFunctions.js (hunk 3 above)
Comment (kgajriaAgarwal) [reply src/Helpers/Reducer/filterFunctions.js]: changed!

File: src/Pages/NotesPage/CreateNote.js (hunk 4 above)
Comment (kgajriaAgarwal) [reply src/Pages/NotesPage/CreateNote.js]: removed!

File: src/Pages/Shared/RightBar/RightBar.js (hunk 6 above)
Comment (kgajriaAgarwal) [reply src/Pages/Shared/RightBar/RightBar.js]: removed !

File: src/Pages/Shared/NoteCard/NoteCard.js (hunk 5 above)
Comment (kgajriaAgarwal) [reply src/Pages/Shared/NoteCard/NoteCard.js]: removed

**Pull request merged**